Zend/tests/pattern_matching/is/range.phpt

```diff
@@ -0,0 +1,41 @@
+--TEST--
+Or pattern
+--FILE--
+<?php
+
+var_dump(0 is 0 ..< 10);
+var_dump(5 is 0 ..< 10);
+var_dump(10 is 0 ..< 10);
+var_dump(11 is 0 ..< 10);
+var_dump(0 is 0 ..= 10);
+var_dump(5 is 0 ..= 10);
+var_dump(10 is 0 ..= 10);
+var_dump(11 is 0 ..= 10);
+
+var_dump('a' is 'a' ..< 'c');
+var_dump('b' is 'a' ..< 'c');
+var_dump('c' is 'a' ..< 'c');
+var_dump('d' is 'a' ..< 'c');
+var_dump('a' is 'a' ..= 'c');
+var_dump('b' is 'a' ..= 'c');
+var_dump('c' is 'a' ..= 'c');
+var_dump('d' is 'a' ..= 'c');
+
+?>
+--EXPECT--
+bool(true)
+bool(true)
+bool(true)
```

Comment:
Shouldn't this be false?  It's coming from (I think) the `var_dump(11 is 0 ..< 10);`, which I would read as "0-10, not including 10".  (Using `..=` would mean "including 10.)

Reply:
Yeah, this looks wrong.